tests/test_parsig_api.py

```diff
@@ -131,10 +156,10 @@ def test_parse_multiple_instructions2(self):
         self.assertEqual(result, expected)
 
     def test_parse_multiple_instructions3(self):
-        sig = "take two tablets of benadryl every two days and then 1 tablet every week for 1 month and than 1 tablet every month"
+        sig = "take two tablets of benadryl every two days and then 1 tablet q.7.d for 1 month and than 1 tablet every month"
         first_expected = StructuredSig(drug="benadryl", form="tablet", strength=None, frequencyType="Day", interval=2,
```

Comment:
choose more indicative name - the first of what?

Reply:
is it fixed? (I did not see the changes in the code)

Reply:
The naming wasn't done by me I only returned the content to its previous status